SmartPlaces.Facilities/products/Topology/src/Topology.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<AssemblyName>$(MSBuildProjectName)</AssemblyName>
+		<RootNamespace>$(MSBuildProjectName)</RootNamespace>
+		<CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\..\..\.config\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
```

Comment:
Can you actually get any of these rules to trigger? I'm not having any success

Reply:
It was working earlier. But now it isn't... these paths... :-(

Reply:
It's working in the libs